clusterloader2/testing/load/daemonset.yaml

```diff
@@ -28,14 +28,10 @@ spec:
             memory: "10M"
       priorityClassName: daemonset-priorityclass-0 # Name is autogenerated, hence the -0 prefix.
       terminationGracePeriodSeconds: 1
-      # Add not-ready/unreachable tolerations for 15 minutes so that node
```

Comment:
We're still adding tolerations (just not for 15minutes) so I don't see a reason for deleting the comment.

If what you wrote in the PR description is true (and daemonset controller adds these tolerations) then we can remove the whole tolerations section, right?

Reply:
Right